GitUI/GitUICommands.cs

```diff
@@ -670,19 +670,18 @@ public bool StartFixupCommitDialog(IWin32Window owner, GitRevision revision)
 
         public bool StartCommitDialog(IWin32Window owner, bool showOnlyWhenChanges)
         {
+            var form = new FormCommit(this);
```

Comment:
I'd prefer to wrap whole method body into `using` block to avoid manual handling of form lifetime and disposing it.